src/Parameters/InsertDocumentParameters.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of littleredbutton/bigbluebutton-api-php.
+ *
+ * littleredbutton/bigbluebutton-api-php is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * littleredbutton/bigbluebutton-api-php is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with littleredbutton/bigbluebutton-api-php. If not, see <http://www.gnu.org/licenses/>.
+ */
+namespace BigBlueButton\Parameters;
+
+/**
+ * @method string getMeetingID()
+ * @method $this setMeetingID(string $id)
+ */
+final class InsertDocumentParameters extends MetaParameters
+{
+    /**
+     * @var string
+     */
+    protected $meetingID;
+
+    /**
+     * @var array
+     */
+    protected $presentations = [];
+
+    public function __construct(string $meetingID)
+    {
+        $this->meetingID   = $meetingID;
+    }
+
+    /**
+     * @return CreateMeetingParameters
+     */
+    public function addPresentation(string $url, string $filename, ?bool $downloadable = null, ?bool $removable = null): self
+    {
+        $this->presentations[$url] = [
+            'filename'     => $filename,
+            'downloadable' => $downloadable,
+            'removable'    => $removable,
+        ];
+
+        return $this;
+    }
+
+    /**
+     * @return array
```

Comment:
I think we can remove this doc. Right?

Reply:
Yes, on adding return types I am planning to get rid of all redundant doc blocks anyway ^^ .

Reply:
The methode was removed in 70574d46657ddc328505156a3e46e5b152e01896
@sualko Was that intended?

Reply:
Yes, it was. If we want to expose that method, it should have a consistent interface and not a deep array as return value. And that it is not worth atm.